src/App.tsx

```diff
@@ -15,36 +15,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabet = 'alphabet',
+  Length = 'length',
+}
+
+interface SortParams {
+  sortField: string;
+  isReverse: boolean;
+}
+
+const getPreparedGoods = (goods: string[], sorter: SortParams) => {
+  const { sortField, isReverse } = sorter;
+  const prepGoods = [...goods];
+
+  if (sortField === SortType.Alphabet) {
+    prepGoods.sort((a, b) => a.localeCompare(b));
+  }
+
+  if (sortField === SortType.Length) {
+    prepGoods.sort((a, b) => a.length - b.length);
+  }
+
+  if (isReverse) {
+    prepGoods.reverse();
+  }
+
+  return prepGoods;
+};
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
The initial state of `sortField` is set to an empty string. It would be more appropriate to initialize it with one of the `SortType` enum values to ensure consistency with the sorting logic.